nocrypto-asymmetric.opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+homepage:     "https://github.com/mirleft/ocaml-nocrypto"
+dev-repo:     "git+https://github.com/mirleft/ocaml-nocrypto.git"
+bug-reports:  "https://github.com/mirleft/ocaml-nocrypto/issues"
+doc:          "https://mirleft.github.io/ocaml-nocrypto/doc"
+authors:      ["David Kaloper <david@numm.org>"]
+maintainer:   "David Kaloper <david@numm.org>"
+license:      "ISC"
+tags:          [ "org:mirage" ]
+
+build: [
+  ["ocaml" "pkg/pkg.ml" "build" "--pkg-name" "%{name}%"
+           "--pinned" "%{pinned}%" "--tests" "false"]
+  ["ocaml" "pkg/pkg.ml" "build" "--pkg-name" "%{name}%"
+           "--pinned" "%{pinned}%" "--tests" "true"] {with-test}
+  ["ocaml" "pkg/pkg.ml" "test" "--pkg-name" "%{name}%"] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.02.0"}
+  "ocamlfind" {build}
+  "ocamlbuild" {build}
+  "topkg" {build}
+  "ppx_deriving" {build}
+  "ppx_sexp_conv"
+  "ounit" {with-test}
+  "cstruct" {>="3.0.0"}
+  "nocrypto"
+  "zarith"
+  "sexplib"
+  ("mirage-no-xen" | "mirage-xen" & "zarith-xen")
+  ("mirage-no-solo5" | "mirage-solo5" & "zarith-freestanding")
+]
+
+conflicts: [
+  "topkg" {<"0.9.1"}
+  "sexplib" {="v0.9.0"}
```

Comment:
I think these can be inlined as dependencies rather than conflicts, since they aren't depopts.